common/nicknames/wc_nicknames.txt

```diff
@@ -87,7 +88,6 @@ nick_the_bloodletter = {}
 nick_the_wretched = { is_bad = yes }
 nick_the_dead = {}
 nick_the_unkillable = {}
-nick_the_harmonious = {}
```

Comment:
Why did you delete that?

Reply:
It was in there twice. Someone added it a second time while I was working on it (they also changed a bunch of other characters in pandaria and i had to revert some of them)